src/components/constant.js

```diff
@@ -32,3 +32,131 @@ export const Ginger = "/ginger/";
 export const Garlic = "/garlic/";
 export const Pepper = "/pepper/";
 export const Curry = "/curry/";
+
+//an array of objects for items on row1
+const Row1Items = [
+{
+    name: "Egg crates",
+    cost: "#1.200",
+    quantityAvailable: "1000",
+    location: "Owerri Imo State",
+    email: "myemail@example.com"
+},
+{
+    name: "Palm oil",
+    cost: "#10,000",
+    quantityAvailable: "500 gallons",
+    location: " Ondo state",
+    email: "myemail@example.com"
+},
+{
+    name: "Onion bags",
+    cost: "#11,500",
+    quantityAvailable: "100",
+    location: "Zaria, Kaduna state",
+    email: "myemail@example.com"
+},
+{
+    name: "Heritage Vegetable Oil ",
+    cost: "#1.500",
+    quantity: "600",
+    location: "Shoprite, Owerri, Imo state",
+    email: "myemail@example.com"
+}
+]
+
+//an array of objects for items on row2
+const Row2Items = [
+    {
+        name: "Abu's farm",
+        cost: "#1.350",
+        quantity: "1000",
+        location: "Niger state",
+        email: "myemail@example.com" 
+    },
+    {
+        name: "Green banana",
+        cost: "#500",
+        quantity: "400 bunches",
+        location: "Edo state",
+        email: "myemail@example.com"
+    },
+    {
+        name: "Bags of rice",
+        cost: "#15,500",
+        quantity: "87",
+        location: "Gurara, Niger state",
+        email: "myemail@example.com"
+    },
+    {
+        name: "Egg crates",
+        cost: "#4,500",
+        quantity: "50",
+        location: "",
+        email: "myemail@example.com"
+    }
+]
+
+//array of objects for row3 items
+const row3Items = [
+    {
+        name: "Lettuses",
+        cost: "#750",
+        quantity: "550",
+        location: "",
+        email: "myemail@example.com"
+    },
+    {
+        name: "Pig farm",
+        cost: "#5000",
+        quantity: "150",
+        location: "",
+        email: "myemail@example.com"
+    },
+    {
+        name: "Carrot",
+        cost: "#3,500",
+        quantity: "3000",
+        location: "",
+        email: "myemail@example.com"
+    },
+    {
+        name: "Poultry",
+        cost: "#1,299",
+        quantity: "2500",
+        location: "",
+        email: "myemail@example.com"
+    }
+]
+
+//array of objects for row4 items
+const row4Items = [
+    {
+        name: "Tomatoes",
+        cost: "#10,000",
+        quantity: "500  baskets",
+        location: "benue state",
+        email: "myemail@example.com"
+    },
+    {
+        name: "Bag of beans",
+        cost: "#15,000",
+        quantity: "150 bags",
+        location: "Niger state",
+        email: "myemail@example.com"
+    },
+    {
+        name: "Ogbono (Docanut)",
+        cost: "#2,000",
+        quantity: "30bags",
+        location: "Imo state",
+        email: "myemail@example.com"
+    },
+    {
+        name: "Ripe banana",
+        cost: "#500",
+        quantity: "200 bunches",
+        location: "Ore, ondo state",
+        email: "myemail@example.com"
+    }
+]
```

Comment:
Very good, this is a temporary evil until we have the database hooked up properly

Reply:
ok